MoEngagePluginBase/MoEngageHybridToNativeConfig.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MoEngageHybridToNativeConfig.swift
+//  MoEngagePluginBase
+//
+//  Created by UdayKiran Naik on 02/07/24.
+//
+
+import Foundation
```

Comment:
Do we need to create a new class for this ? Can't we reuse Any of the existing classes ?